scripts.d/support/install-setup

```diff
@@ -4,6 +4,41 @@ source $(dirname ${BASH_SOURCE[0]})/install-setup-common
 
 AUTO=0
 
+function desktop-icon()
```

Comment:
This function should be moved into the shared routines files:
`install-setup-common`